cmd/root.go

```diff
@@ -19,9 +19,10 @@ var config struct {
 
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
-	Use:   "pvc-autoresizer",
-	Short: "PVC Autoresizer",
-	Long:  `pvc-autoresizer is an automatic volume resizer that edits PVCs if they have less than the specified amount of free filesystem capacity.`,
+	Use:          "pvc-autoresizer",
+	Short:        "PVC Autoresizer",
+	Long:         `pvc-autoresizer is an automatic volume resizer that edits PVCs if they have less than the specified amount of free filesystem capacity.`,
+	SilenceUsage: true,
```

Comment:
This should be set in `RunE`, or you never get usage message even when the usage is wrong.
Please see our other products.